app/web/components/constants.ts

```diff
@@ -1,10 +1,5 @@
 export const CANCEL_UPLOAD = "Cancel upload";
```

Comment:
What about the converting the constants in this file? Or are you planning to do this separately?

Reply:
Maybe separately